### ISSUE TITLE
Pretrained model for Flickr30k

### ISSUE THREAD

**Issue by drigoni** [jnhwkim/ban-vqa#47]
Dear Authors,

the link to download the pre-trained model for Flickr30k no longer works. Could you please update it again?
Link not working: https://drive.google.com/uc?export=download&id=1xiVVRPsbabipyHes25iE0uj2YkdKWv3K

Davide

**Comment by jnhwkim** [comment]
@jaesuny do you have access to this?

**Comment by jnhwkim** [comment]
Regrettably, the link is no longer accessible, and I cannot perform maintenance due to the loss of the materials. :(
